strongbox-aql/pom.xml

```diff
@@ -40,22 +40,108 @@
+                    </dependency>
+                    <dependency>
+                        <groupId>org.jtwig</groupId>
+                        <artifactId>jtwig-core</artifactId>
+                        <version>5.86.1.RELEASE</version>
+                        <exclusions>
+                            <exclusion> 
+                                <groupId>com.googlecode.concurrentlinkedhashmap</groupId>   
+                                <artifactId>concurrentlinkedhashmap-lru</artifactId>    
+                            </exclusion>
+                            <exclusion>
+                                <groupId>com.google.guava</groupId>
+                                <artifactId>guava</artifactId>
+                            </exclusion>
+                        </exclusions>   
+                    </dependency>
+                    <dependency>
+                        <groupId>org.codehaus.groovy</groupId>
+                        <artifactId>groovy-all</artifactId>
+                        <version>${version.groovy}</version>
+                        <scope>runtime</scope>
+                    </dependency>
+                </dependencies>
+            </plugin>
             <plugin>
                 <groupId>org.antlr</groupId>
                 <artifactId>antlr4-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>antlr</id>
+                        <id>aql-antlr-java</id>
+                        <phase>generate-sources</phase>
                         <goals>
                             <goal>antlr4</goal>
                         </goals>
                         <configuration>
+                            <arguments>
+                                <argument>-package</argument>
+                                <argument>org.carlspring.strongbox.aql.grammar</argument>
+                            </arguments>
+                            <sourceDirectory>${project.build.directory}/antlr4</sourceDirectory>
                             <visitor>true</visitor>
                             <outputDirectory>${project.build.directory}/generated-sources/antlr4/org/carlspring/strongbox/aql/grammar</outputDirectory>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>aql-grammar-artifact</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <classesDirectory>${project.build.directory}/antlr4</classesDirectory>
+                            <includes>
+                                <include>AQL.g4</include>
```

Comment:
I think this file's name should be lowercase.

Reply:
yeah, but can we leave this for followup PR? this will cause a lot of renamings